packages/playwright/src/executors/playwright/playwright.ts

```diff
@@ -81,10 +81,21 @@ export async function playwrightExecutor(
 
   const args = createArgs(options);
   const p = runPlaywright(args, context.root);
+  p.stdout.on('data', (message) => {
+    process.stdout.write(message);
+  });
+  p.stderr.on('data', (message) => {
+    process.stderr.write(message);
+  });
 
   return new Promise<{ success: boolean }>((resolve) => {
     p.on('close', (code) => {
-      resolve({ success: code === 0 });
+      if (code !== 0) {
+        console.error(`Playwright exited with code ${code}`);
```

Comment:
do we need this? Since the executor fails when it's not 0 that information is already conveyed.

Reply:
removed